packages/fxa-content-server/tests/functional/send_sms.js

```diff
@@ -112,6 +113,38 @@ const suite = {
     );
   },
   tests: {
+    'displays expected header with `syncPhone` experiment group': function() {
+      return this.remote
+        .then(
+          openPage(SEND_SMS_HEADER_URL, {
```

Comment:
I think `forceExperiment` is covered by `SEND_SMS_URL` (L26)?

Reply:
@vbudhram maybe we did this an atypical way, but since `forceExperiment` was the same between the two test additions, we left that as a parameter in `SEND_SMS_HEADER_URL` (I think @jodyheavener meant to reference the const with `HEADER`)

Reply:
Although @jodyheavener clearly we have something incorrect as those tests are failing in CI ;)

Reply:
> I think forceExperiment is covered by SEND_SMS_URL (L26)?

Oh, yea that works too. CI might be failing because the 2nd param in `openPage` requires a selector to check when page is ready instead of query object.